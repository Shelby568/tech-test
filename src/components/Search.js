import React, { useState } from 'react';

const Search = props => {
    return (
        <>
        <input className="searchbox" type="text"/>
        </>
    )
};

export default Search;