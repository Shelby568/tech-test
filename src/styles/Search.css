.searchform {
    display: flex;
    flex-direction: row;
    justify-content: center;
}

.searchbox {
    width: 20%;
    height: 40px;
    border-radius: 4px;
    font-size: 20px;
    padding: 0 10px;
}

.submit {
    border-radius: 4px;
    background-color:#B22222;
    color: white;
    font-weight: bold;
    font-size: 1.15rem;
    width: 75px;
}