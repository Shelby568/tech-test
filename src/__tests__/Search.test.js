import React from 'react';
import { render } from "@testing-library/react";
import Search from '../components/Search';

describe('Search', () => {
    const { asFragment } = render(<Search />)

    it('make sure component renders correctly', () => {
        expect(asFragment()).toMatchSnapshot();
    });
});