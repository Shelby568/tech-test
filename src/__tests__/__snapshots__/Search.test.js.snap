// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search make sure component renders correctly 1`] = `
<DocumentFragment>
  <input
    class="searchbox"
    type="text"
  />
</DocumentFragment>
`;
